components/button/index.css

```diff
@@ -125,6 +125,43 @@ governing permissions and limitations under the License.
   }
 }
 
+.spectrum-Button--overBackground,
+.spectrum-Button--overBackground.spectrum-Button--quiet {
```

Comment:
Is this extra selector required because `.spectrum-Button--quiet` overrides some of this?

Reply:
Removed in latest commits.